### PR TITLE
Active Learning Tutorial - ECC-5455

### PR DESCRIPTION
**Pull request by rpietzsch**
- active learning
- active learning
- Activelearning
- active learning
- review results
- fix page icon refine screenshot crop
- add UI icons to the text. use classic tag names.

### CODE REVIEW

File: docs/build/active-learning/index.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+-   [Usage](#usage)
+-   [Start the learning dialog](#start-the-learning-dialog)
+-   [Creating an automatic link rule](#creating-an-automatic-link-rule)
+-   [Add property paths for both entities](#add-property-paths-for-both-entities)
+
+## Usage
+
+Define properties to compare between entities of the selected datasets.
+
+## Start the learning dialog
```
Comment (seebi) [inline docs/build/active-learning/index.md:27]: IMHO this is not enough to give the user a starting point.

File: docs/build/active-learning/index.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+Create a linking rule and setup source and target datasets as well as the linking property that should be yielded.
+Start the learning dialog by clicking the “Learning” tab in the linking view.
+
+## Creating an automatic link rule
+
+-   Choose properties to compare.
+    Select from the suggestions or search them by specifying property paths for both entities.
+
+    ![image](22.2-Suggestion.png){ class="bordered" }
```
Comment (seebi) [inline docs/build/active-learning/index.md:37]: FYI - I introduced a new class in `extra.css` to be reused where needed.

**Review by seebi — CHANGES_REQUESTED** [review]
looks good - only some additions needed.
small fixes I did myself.

File: docs/build/active-learning/index.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+-   [Introduction](#introduction)
+-   [Usage](#usage)
+-   [Start the learning dialog](#start-the-learning-dialog)
+-   [Creating an automatic link rule](#creating-an-automatic-link-rule)
+-   [Add property paths for both entities](#add-property-paths-for-both-entities)
+
+## Usage
+
+Define properties to compare between entities of the selected datasets.
```
[the remark this answers is not in the document]
Comment (rpietzsch) [reply docs/build/active-learning/index.md]: I agree, section extended.

File: docs/build/active-learning/index.md (hunk 1 above)
Comment (rpietzsch) [reply docs/build/active-learning/index.md:36]: extended.

File: docs/build/active-learning/index.md (hunk 2 above)
Comment (rpietzsch) [reply docs/build/active-learning/index.md:51]: thank you 🙌

**Comment by rpietzsch** [comment]
@seebi pls. re-check.

**Review by seebi — APPROVED** [review]

**Pull request merged**